### PR TITLE
[RFC] News

### PR DESCRIPTION
**Pull request by KillTheMule**
Continuation of https://github.com/neovim/neovim.github.io/pull/151 with the goal of getting something release-ready or at least provide a nice starting point for the next newsletter.

I went with the attitude "heck I'll just do it the way I want", which meant removing quite a bit of stuff. @timeyyy I hope you're ok with this, thanks for your initial work.

I'm very open to suggestions and criticism. More than that, I'll definitely need help on the following:

* What notable PRs were merged since the 0.2 release?
* What notable stuff is upcoming?
* What else should be in it?

I'm happy about any pointers, so feel free to just provide a keyword or so, I'll find the PRs and read up on them for sure :)

### CODE REVIEW

**Comment by KillTheMule** [comment]
@justinmk  I've thought it would be nice to mention new contributors. Now, the list of all new contributors is probably a bit too long, do you have any idea about a criterion for inclusion?

**Comment by nelstrom** [comment]
@KillTheMule thanks for taking the initiative on this 😃

File: _posts/2017-05-02-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+- Windows TUI (https://github.com/neovim/neovim/pull/6315)
+- AppImage packages for Linux (https://github.com/neovim/neovim/pull/6638)
+
+### 0.2.1-dev for Developers
+- Static Analysis with PVS (???)
+- Externalized tabline (???)
+- Eval.c refactor (https://github.com/neovim/neovim/pull/5119)
+- ui_events now reported in api metadata (???)
+- First usage of lua to implement core functionality
+(https://github.com/neovim/neovim/pull/67890)
```
Comment (cweagans) [inline _posts/2017-05-02-newsletter.md]: This link is wrong. https://github.com/neovim/neovim/pull/6789 maybe?

Comment (KillTheMule) [reply _posts/2017-05-02-newsletter.md]: Fixed, thanks.

**Comment by timeyyy** [comment]
All good, thanks for putting steam back into this!

**Comment by justinmk** [comment]
@KillTheMule I try to call out people by name when mentioning specific work. I don't have any objective criteria for this.

**Comment by justinmk** [comment]
> What else should be in it?

I have been tracking notes in:

- https://github.com/neovim/neovim.github.io/issues/148
- https://github.com/neovim/neovim.github.io/pull/151#issuecomment-298629950
- https://github.com/neovim/neovim.github.io/pull/151#issuecomment-300470068

**Comment by KillTheMule** [comment]
Ah very good, thanks. I've seen your comments in #151 and started to add that content, but I missed #148.

**Comment by KillTheMule** [comment]
Ok, I went through those and added the things I found relevant. I left out the Racket client, there have been quite some additions to the clients on the related projects page, and I have not seen a reason to single out this one. Anyone thinks differently?

Next step will be to search though the commits from 0.2 forward to see if there's something I should mention.

As far as upcoming things go, what about the lua parser? Can someone explain the significance of it?

Oh, yeah, I should check out the 0.2.1 milestone and see what's planned and what I should mention.

**Comment by KillTheMule** [comment]
Yeah I had 2 entries there, but I thought I'd fixed it... thanks!

**Comment by bfredl** [comment]
But it could mention "colored commandline" itself of course, which will work for `input()` already in 0.2.1

**Comment by KillTheMule** [comment]
This is nearing some sort of completion from my side.

@ZyX-I @bfredl @justinmk @jamessan Would you kindly check if there's something missing content-wise? In particular the `upcoming` list seems a bit barebone to me, so maybe you have something  that I should add. I also have the following notes

- ui_events now reported in api metadata (???)
- externalized 'cursorline'/'cursorcolumn' will allow 'cursorline' without a performance penalty (???)

but somehow can't find related PRs for those items. Can you point me towards something?

@justinmk Could you imagine merging this after a bit more polish? Are you missing something? Let me know what you think :)

File: _posts/2017-05-02-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+Neovim is a fork of the venerable text-editor vim, focused on extensibility and
+usability. It is not a rewrite but a continuation and extension of Vim. Many
+clones and derivatives exist, some very clever—but none are Vim. Neovim is built
+for users who want the good parts of Vim, and more. See a list of differences
+via [:help vim-differences](https://neovim.io/doc/user/vim_diff.html).
+
+### Participating
+
+If you are interested in contributing to neovim, visit [the github
```
Comment (ZyX-I) [inline _posts/2017-05-02-newsletter.md]: Should not “Neovim” start with a capital here and in some places below?

File: _posts/2017-05-02-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+  - **@aignas** improved path handling on windows (https://github.com/neovim/neovim/pull/7349)
+- For Linux users:
+  - **@AdnoC** contributed AppImage packages for Linux (https://github.com/neovim/neovim/pull/6638)
+
+
+
+### 0.2.1-dev for Developers
+- **@ZyX-l** enabled static Analysis with PVS (https://github.com/neovim/neovim/pull/6493)
+- Thanks to **@dzhou121**, external UIs can now draw the tabline (https://github.com/neovim/neovim/pull/6583)
+- Despite it being 100% positively impossible, **@ZyX-l** managed to refactor `eval.c` (https://github.com/neovim/neovim/pull/5119)
```
Comment (ZyX-I) [inline _posts/2017-05-02-newsletter.md]: I did not refactor the entire `eval.c`, I only moved a selected number of functions to another location and refactored them. Bram did more splitting.

File: _posts/2017-05-02-newsletter.md (hunk 2 above)
Comment (KillTheMule) [reply _posts/2017-05-02-newsletter.md]: Right, thanks!

File: _posts/2017-05-02-newsletter.md (hunk 3 above)
Comment (KillTheMule) [reply _posts/2017-05-02-newsletter.md]: Ah ok. I left that point out then, hope that's ok for you.

**Comment by tjdevries** [comment]
selfishly hoping you might include this: https://github.com/neovim/neovim/pull/6236 😜 

Programmatic way to get information about mappings (and gives more information than previous methods could have, like `maparg()`).

**Comment by KillTheMule** [comment]
Sure, thanks for the hint :)

**Comment by bfredl** [comment]
Upcoming features could also mention extended marks neovim/neovim#5031 and buffer change notifications neovim/neovim#5269

**Comment by bfredl** [comment]
Also LSP neovim/neovim#6856 if it doesn't already

**Comment by KillTheMule** [comment]
Thanks @bfredl!

**Comment by justinmk** [comment]
I do think the rkt client should be mentioned, and also want to call out https://github.com/alexgenco/neovim-ruby for being a very well-maintained project.

**Comment by KillTheMule** [comment]
Ok, I went through the revisions of the related projects page and wrote up things a bit. Other than adding the stats about doc change I wouldn't know what else to change. I went over the language and improved things a bit, but it's now in a state I like. I'm not a native english speaker, so anyone crossreading would be appreciated!

All suggestions still welcome, of course :)

File: _posts/2017-05-02-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+
+The wiki page of [related
+projects](https://github.com/neovim/neovim/wiki/Related-projects) has seen quite
+some additions, check out the full changes [here](https://github.com/neovim/neovim/wiki/Related-projects/_compare/7ce8559...e2c051fd85226ff6a67c67f9de55ad9039aa2948).
+
+
+### API clients
+
+API clients are at the heart of Neovim's architecture, and significantly improve developer experience. Write a plugin in
+any language you want! Since the last newsletter, we've seen the addition of another [C++](https://github.com/DaikiMaekawa/neovim.cpp) client, one for [Elixir](https://github.com/dm1try/nvim), and a [Racket](https://gitlab.com/HiPhish/neovim.rkt) client. The [node client](https://github.com/neovim/node-client) got a an overhaul and a newmaintainer. Thanks **@billyvg** for taking over! While not totally new, a special shoutout goes to the [ruby client](https://github.com/alexgenco/neovim-ruby) for being very well-maintained. Thanks, **@alexgenco**!
```
Comment (davidgranstrom) [inline _posts/2017-05-02-newsletter.md]: newmaintainer -> new maintainer

Comment (KillTheMule) [reply _posts/2017-05-02-newsletter.md]: Fixed, thanks!

File: _posts/2017-05-02-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+
+It features windows support, various terminal improvements, full API
+documentation for developers and a host of smaller fixes and improvements. A small highlight: A long standing issue with using `<Ctrl-h>` has been [fixed](https://github.com/neovim/neovim/wiki/FAQ#my-ctrl-h-mapping-doesnt-work).
+
+See
+the [release page](https://github.com/neovim/neovim/releases/tag/v0.2.0) for a
+detailed summary and download opportunities.
+
+### 0.2.1-dev for Users
+- **@bfredl** brought us window specific background color (https://github.com/neovim/neovim/pull/6597)
```
Comment (bfredl) [inline _posts/2017-05-02-newsletter.md]: it's actually window highlighting in general, not just background: neovim/neovim#6700

Comment (KillTheMule) [reply _posts/2017-05-02-newsletter.md]: Fixed, thanks!

**Comment by KillTheMule** [comment]
I've updated the link bfredl mentioned, I've also put in the doc stats I wanted. I also squashed, but I left @timeyyy's commits alone because I don't know how to deal with 2 authors for 1 PR. Maybe just leave 1 of his commits in? Let me know if I should do something specific.

@justinmk: When/if you merge this, you might want to rerun the stats command, they're C&Pable. You will also need to change the file name and the permalink at the top of the post to the corresponding date. Or just let me know in time so I'll do it :)

**Comment by KillTheMule** [comment]
Added 2 merged PRs of @teto.

File: _posts/2017-05-02-newsletter.md [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+is certainly
+
+### Release 0.2
+
+It features windows support, various terminal improvements, full API
+documentation for developers and a host of smaller fixes and improvements. A small highlight: A long standing issue with using `<Ctrl-h>` has been [fixed](https://github.com/neovim/neovim/wiki/FAQ#my-ctrl-h-mapping-doesnt-work).
+
+See
+the [release page](https://github.com/neovim/neovim/releases/tag/v0.2.0) for a
+detailed summary and download opportunities.
```
Comment (justinmk) [inline _posts/2017-05-02-newsletter.md]: This should really be condensed into the newsletter, that's what people want.

Comment (KillTheMule) [reply _posts/2017-05-02-newsletter.md]: Ah, I figured it's a thing of the past, and the newsletter should concentrate on the happenings after that :) No matter, I'll integrate it.

**Comment by KillTheMule** [comment]
I integrated the 0.2 release notes (it's too long now for a newsletter imho, but I did not have it in me not to mention some of the contributions), and I restructured the text a bit as a consequence.

Do we have a reference for the man.vim changes?

Is there a good link to the new API documentation? Would that be https://neovim.io/doc/user/msgpack_rpc.html#RPC?

**Comment by bfredl** [comment]
No, https://neovim.io/doc/user/api.html

**Comment by KillTheMule** [comment]
Thanks :)

**Comment by KillTheMule** [comment]
I went backward though the commits up to 20171105, and added those that stood out to me to the newsletter.

File: _posts/2017-05-02-newsletter.md [hunk 7]
```diff
@@ -0,0 +1,202 @@
...
+### The Architecture of Neovim.
+
+Some students have studied the Architecture of Neovim and published an
+"analytical essay" about their findings. Be sure to
+[have a look](https://delftswa.gitbooks.io/desosa-2017/content/neovim/chapter.html)
+if you are interested in the bigger picture.
+
+### Vimcasts and Modern Vim
+
+Drew Neil of [vimcasts](http://vimcasts.org) fame has recorded [10 casts](http://vimcasts.org/blog/2017/06/dive-into-neovim/) about Neovim! You can watch the first 3 for free now, while the rest needs some sort of subscription. He even began work on a successor to his praised book [Practical Vim](https://pragprog.com/book/dnvim2/practical-vim-second-edition) called [Modern Vim](http://vimcasts.org/blog/2017/05/working-title-modern-vim/), and plans to dedicate about a third of it to Neovim-only features (with the rest of the book still being fully relevant to Neovim users).
```
Comment (nelstrom) [inline _posts/2017-05-02-newsletter.md]: > Drew Neil ... has recorded 10 casts about Neovim! You can watch the first 3 for free now, while the rest needs some sort of subscription.

I'm going to be publishing all 10 of those episodes on the Vimcasts site over the coming months. I think people might get mad if you link them to the Thoughtbot Prime page, because the content is behind a paywall (save for the first 3 episodes). Instead, it might be best to just mention that I'm making videos again and will be covering some of Neovim's features. 

> ...plans to dedicate about a third of it to Neovim-only features (with the rest of the book still being fully relevant to Neovim users)

I had to read the bit in parentheses a couple of times to make sense of it. I can understand what you're saying and it's correct, but it's a bit awkward. I'd maybe rephrase it along these lines: "Most of the book is suitable for Vim 8 and Neovim users, but about 1/3rd of the book covers Neovim-specific functionality."

**Comment by justinmk** [comment]
another cool UI concept: https://github.com/jeanguyomarch/eovim/

File: _posts/2017-05-02-newsletter.md (hunk 7 above)
Comment (KillTheMule) [reply _posts/2017-05-02-newsletter.md]: Thanks for your comments!

**Comment by KillTheMule** [comment]
Went back until 20170730.

**Comment by KillTheMule** [comment]
I further backtracked to https://github.com/neovim/neovim/commits/master?after=e35a66d39670d19d50bf732b3058c12e33c77440+559. I'm just adding things that catch my eye, so if anyone notices a PR I should mention but don't, please let me know. I see a lot of work by @ZyX-I , bt I'm not sure what to mention. 

Other than that, I'm away till the weekend, so I'll probably be quiet till then ;)

File: _posts/2017-05-02-newsletter.md [hunk 8]
```diff
@@ -0,0 +1,208 @@
...
+  - GUIs now should use the `FocusGained` event instead of sending the `<FocusGained>` pseudokey as implemented by **@justinmk** (https://github.com/neovim/neovim/pull/7221)
+ - UI's can get menus from **@teto's** `menu_get` https://github.com/neovim/neovim/pull/6322
+  
+### The not-so-near future: Upcoming
+
+For an overview of planned features, goals and ideas for Neovim head to the [road
+map](https://neovim.io/roadmap/). Some noteworthy upcoming PRs are:
+
+- [Floating Windows in the Terminal](https://github.com/neovim/neovim/pull/6619) by **@bfredl**
+- [Externalized cmdline](https://github.com/neovim/neovim/pull/6162) by **@dzhou121**
```
Comment (bfredl) [inline _posts/2017-05-02-newsletter.md]: This is now in 0.2.1

File: _posts/2017-05-02-newsletter.md [hunk 9]
```diff
@@ -0,0 +1,207 @@
...
+ - External UI's can now draw the command line (**@dzhou121**, **@bfredl**, https://github.com/neovim/neovim/pull/6162)
+ - Inccomand now works with leading modifiers like `keeppattern` (**@jamessan**, https://github.com/neovim/neovim/pull/6967)
+  
+### The not-so-near future: Upcoming
+
+For an overview of planned features, goals and ideas for Neovim head to the [road
+map](https://neovim.io/roadmap/). Some noteworthy upcoming PRs are:
+
+- [Floating Windows in the Terminal](https://github.com/neovim/neovim/pull/6619) by **@bfredl**
+
```
Comment (bfredl) [inline _posts/2017-05-02-newsletter.md]: leftover blank :)

File: _posts/2017-05-02-newsletter.md [hunk 10]
```diff
@@ -0,0 +1,207 @@
...
+- First usage of lua to implement core functionality (**@ZyX-l**, https://github.com/neovim/neovim/pull/6789)
+- Call lua directly from the API (**@bfredl**, https://github.com/neovim/neovim/pull/6704)
+- Programmatically get information about mappings via  `get_keymap` (**@tjdevries**, https://github.com/neovim/neovim/pull/6236)
+- `menu_get` now pretty-prints special chars, making it possible to feed its results back into e.g. `nvim_input` (**@teto**, **@KillTheMule**, https://github.com/neovim/neovim/pull/7340)
+ - You can now retrieve highlights `using nvim_get_hl_by_name/by_id` (**@teto**, https://github.com/neovim/neovim/pull/7082)
+ - `bufhl` can now be used to create new highlighting groups (**@bfredl**, https://github.com/neovim/neovim/pull/7414)
+ - GUIs now should use the `FocusGained` event instead of sending the `<FocusGained>` pseudokey (**@justinmk**, https://github.com/neovim/neovim/pull/7221)
+ - UI's can get menus from  `menu_get` (**@teto**, https://github.com/neovim/neovim/pull/6322)
+ - External UI's can now draw the command line (**@dzhou121**, **@bfredl**, https://github.com/neovim/neovim/pull/6162)
```
Comment (bfredl) [inline _posts/2017-05-02-newsletter.md]: maybe group with external tabline.

File: _posts/2017-05-02-newsletter.md [hunk 11]
```diff
@@ -0,0 +1,207 @@
...
+  - A newly created `:terminal` will now stay in normal mode (**@nelstrom**, https://github.com/neovim/neovim/pull/6808)
+  - True colors in tmux have been fixed (**@DarkDefender**, https://github.com/neovim/neovim/pull/7100)
+  - Further improvements for vimtutor (**@fmoralesc**, https://github.com/neovim/neovim/pull/7028)
+  - Adjust terminal buffers when using the number column (**@tecywiz121**, https://github.com/neovim/neovim/pull/7440)
+
+
+#### 0.2.1-dev for Developers
+- Neovim sources are now analyzed with PVS (**@ZyX-l**, https://github.com/neovim/neovim/pull/6493)
+- External UIs can now draw the tabline (**@dzhou121**, https://github.com/neovim/neovim/pull/6583)
+- First usage of lua to implement core functionality (**@ZyX-l**, https://github.com/neovim/neovim/pull/6789)
```
Comment (bfredl) [inline _posts/2017-05-02-newsletter.md]: While not wrong, would it not be informative to describe what it does for developers? (lua files can be required from `lua/` dir in runtime and plugins)

File: _posts/2017-05-02-newsletter.md (hunk 10 above)
Comment (bfredl) [reply _posts/2017-05-02-newsletter.md]: or even all UI stuff, but even then the two new externalized features belong together.

Comment (KillTheMule) [reply _posts/2017-05-02-newsletter.md]: Yeah, you're right. I unified the wording, I'll check out a bit of grouping tomorrow.

File: _posts/2017-05-02-newsletter.md (hunk 11 above)
Comment (KillTheMule) [reply _posts/2017-05-02-newsletter.md]: Sure, I just did not realize that :)

**Comment by KillTheMule** [comment]
Ok, I added the improvements suggested by @bfredl, I streamlined the references, I grouped up things a bit, I went through the commit history back to the last release and added what looked significant to me.

I need help understanding those 2 PRs:

- https://github.com/neovim/neovim/pull/6914
- https://github.com/neovim/neovim/pull/6618

The documentation on them is too sparse for me to figure out what they really do and why that would be of significance. Can anyone dumb that down for me?

Other than that, I hope I did not miss anything, and feel like I'm once again nearing completion.

**Comment by bfredl** [comment]
neovim/neovim#6618 is probably too internal to be worth mentioning in the news letter, except possibly that "ui event signatures are now included in the metadata"

neovim/neovim#6914 most importantly allows more commands to follow `endfunction` in certain situations, for instance in a single `nvim_command()` call.

**Comment by KillTheMule** [comment]
> neovim/neovim#6618 is probably too internal to be worth mentioning in the news letter, except possibly that "ui event signatures are now included in the metadata"

In that case I'd leave it out, the newsletter is already much too long imho.

Thanks for your explanation :)

File: _posts/2017-05-02-newsletter.md [hunk 12]
```diff
@@ -0,0 +1,222 @@
...
+### The Architecture of Neovim.
+
+Some students have studied the Architecture of Neovim and published an
+"analytical essay" about their findings. Be sure to
+[have a look](https://delftswa.gitbooks.io/desosa-2017/content/neovim/chapter.html)
+if you are interested in the bigger picture.
+
+### Vimcasts and Modern Vim
+
+Drew Neil of [vimcasts](http://vimcasts.org) fame has recorded [10 casts](http://vimcasts.org/blog/2017/06/dive-into-neovim/) about Neovim, and began work on a successor to his praised book [Practical Vim](https://pragprog.com/book/dnvim2/practical-vim-second-edition) called [Modern Vim](http://vimcasts.org/blog/2017/05/working-title-modern-vim/). Most of the book will be  suitable for Vim 8 and Neovim users, but about 1/3rd of the book will cover Neovim-specific functionality.
```
Comment (nelstrom) [inline _posts/2017-05-02-newsletter.md]: I'd suggest changing a couple of the links:

* instead of http://vimcasts.org/blog/2017/06/dive-into-neovim/ you could link to the [Neovim category](http://vimcasts.org/categories/neovim/). As I publish more screencasts and articles about Neovim they will appear on that page.
* instead of http://vimcasts.org/blog/2017/05/working-title-modern-vim/ you could link to the [Modern Vim page on Pragmatic Bookshelf](https://pragprog.com/book/modvim/modern-vim). Modern Vim has now been published as a beta book.

Comment (KillTheMule) [reply _posts/2017-05-02-newsletter.md]: Good news,thanks for bringing it up!

File: _posts/2017-05-02-newsletter.md [hunk 13]
```diff
@@ -0,0 +1,222 @@
...
+ - You can now define several functions in one `:execute` call (**@ZyX-I**, https://github.com/neovim/neovim/pull/6914)
+ 
+ 
+  
+### The not-so-near future: Upcoming
+
+For an overview of planned features, goals and ideas for Neovim head to the [road
+map](https://neovim.io/roadmap/). Some noteworthy upcoming PRs are:
+
+- Floating Windows in the Terminal (**@bfredl**, https://github.com/neovim/neovim/pull/6619)
```
Comment (bfredl) [inline _posts/2017-05-02-newsletter.md]: "in the Terminal" can be dropped, https://github.com/neovim/neovim/pull/6619 is now also supporting external uis.

**Comment by KillTheMule** [comment]
I updated to adjust for the 0.2.1 release, I also went through the release note to make sure we've got all the new features covered. I did not include most bugfixes, and not all of the changes.

**Comment by XVilka** [comment]
It make sense to rename the file as well.

**Comment by KillTheMule** [comment]
Sure, but I'll leave the up to justinmk, because the name contains the release date :)

**Comment by KillTheMule** [comment]
Thanks!

File: _posts/2017-05-02-newsletter.md [hunk 14]
```diff
@@ -0,0 +1,238 @@
+---
+layout: newsletter
+title: "Newsletter #8 - Open up the Windows"
+category: newsletter
+permalink: /news/2017/05/
```
Comment (jamessan) [inline _posts/2017-05-02-newsletter.md]: Change this to `/news/2017/12/` and rename the file to `_posts/2017-12-15-newsletter.md`.  After updating the stats and squashing as you see fit, I'll merge it.

Comment (KillTheMule) [reply _posts/2017-05-02-newsletter.md]: Ok, I'll find some time this weekend.

File: _posts/2017-05-02-newsletter.md [hunk 15]
```diff
@@ -0,0 +1,238 @@
...
+ - `:terminal` has seen various improvements (**@justinmk**, https://github.com/neovim/neovim/pull/6185)
+   - `findfile`, `:find` and `gf` now work from the terminal (**@tweekmonster**, https://github.com/neovim/neovim/pull/6009)
+   - Shell output is now throttled to maintain terminal responsiveness (**@justinmk**, https://github.com/neovim/neovim/pull/5396)
+    - Arguments for shell commands now work properly with `:term` (**@friedsock**, https://github.com/neovim/neovim/pull/4504)
+ - Numerous improvements to man.vim have been made(**@m-wynn**, **@raichoo**, **@renstrom**, **@justinmk**, https://github.com/neovim/neovim/pull/6693,  https://github.com/neovim/neovim/pull/6766, https://github.com/neovim/neovim/pull/6815, https://github.com/neovim/neovim/pull/3104, https://github.com/neovim/neovim/pull/7405)
+ - Man and help pages now contain a TOC (**@tweekmonster**, https://github.com/neovim/neovim/pull/5169)
```
Comment (wsdjeg) [inline _posts/2017-05-02-newsletter.md]: Maybe use 

```markdown
Man and help pages now contain a TOC (**@tweekmonster**, [neovim/neovim#5169](https://github.com/neovim/neovim/pull/5169))
```

Man and help pages now contain a TOC (**@tweekmonster**, [neovim/neovim#5169](https://github.com/neovim/neovim/pull/5169))

just make text looks nice, same as below.

Comment (KillTheMule) [reply _posts/2017-05-02-newsletter.md]: Good suggestion, thanks. I went with the ultra-short form.

**Comment by jamessan** [comment]
Thanks @KillTheMule!
